arlo_server/models.py

```diff
@@ -278,6 +278,12 @@ class Round(BaseModel):
     audit_boards = relationship("AuditBoard", backref="round", passive_deletes=True)
 
 
+class BallotStatus(str, Enum):
+    NOT_AUDITED = "NOT_AUDITED"
+    AUDITED = "AUDITED"
+    SKIPPED = "SKIPPED"
```

Comment:
not strictly needed yet, but figured i'd throw it in now since we'll be using it later

Reply:
I think this is partially redundant with the presence of the `BallotInterpretation` row, which means the data model could become inconsistent, which is not ideal.

Maybe consider a `SKIPPED` option in the `Interpretation` enum and then you can get rid of this field?

Reply:
I'm with you on having internally consistent data models. Super important.

I think the reason it's more appropriate this way for a few reasons:
- On the product side, audit boards skip a whole ballot, not individual contests.
- According to @mcchilders, an audit board might come across a ballot with no selected contests, in which case they "audit" but don't record any interpretations. I think this is a consequence of the fact that we have no way of knowing which contests are on which ballots when we sample.

Thoughts?
Are there ways we can help prevent this from becoming inconsistent? Constraints?

Reply:
Could also do audited/not audited/not found (if that's clearer in code than "skipped").